src/wp-includes/blocks.php

```diff
@@ -114,12 +114,13 @@ function register_block_script_handle( $metadata, $field_name, $index = 0 ) {
 			substr_replace( $script_path, '.asset.php', - strlen( '.js' ) )
 		)
 	);
-	if ( ! file_exists( $script_asset_path ) ) {
+	if ( false === $script_asset_path ) {
```

Comment:
Also commented on the ticket, but it appears my initial suggestion to use a `false === $script_asset_path` comparison was incorrect. While `realpath()` returns explicit `false` when a file does not exist, `wp_normalize_path()` will return an empty string when passed `false`. Seems an `empty()` check would be more appropriate.

Reply:
Good catch @desrosj 

Changed in https://github.com/WordPress/wordpress-develop/pull/2313/commits/7db1cbaa28566acc5bd394ba8e5ca5ae1b55a272 ✅